contracts/modules/STO/USDTiered/USDTieredSTO.sol

```diff
@@ -41,7 +41,7 @@ contract USDTieredSTO is USDTieredSTOStorage, STO {
         uint256 _rate
     );
     event ReserveTokenMint(address indexed _owner, address indexed _wallet, uint256 _tokens, uint256 _latestTier);
-    event SetAddresses(address indexed _wallet, address indexed _reserveWallet, address[] _usdTokens);
+    event SetAddresses(address indexed _wallet, address indexed treasuryWallet, address[] _usdTokens);
```

Comment:
_treasuryWallet for consistency

Reply:
I don't know how it is showing on my local it is correct.